src/components/location/Location.js

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { Link } from "gatsby"
+import { BsArrowLeft } from "react-icons/bs"
+import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet"
+import Layout from "../layout/Layout"
+import {
+  StyledLocationPageContantTop,
+  StyledLocationPageContantTopInfo,
+  StyledLocationPageCt,
+  StyledLocationPageHeader,
+  StyledLocationPageSecoundContant,
+  StyledLocationPageSecoundContantSinlgeItem,
+  StyledLocationPageSecoundContantToggle,
+} from "./LocationStyle"
+import { MdLocationOn } from "react-icons/md"
```

Comment:
wrong import order, and also, try to keep styles at the last line of imports. it will make imports looks better